editor/components/inserter/index.js

```diff
@@ -39,15 +48,47 @@ class Inserter extends Component {
 		}
 	}
 
+	showInsertionPoint() {
+		const { showInlineInsertionPoint, showInsertionPoint } = this.props;
+
+		if ( this.isInsertingInline() ) {
+			this.setState( { isInline: true } );
```

Comment:
Why do we need local state here?

Reply:
Using local state here prevents the Inline Blocks menu from briefly rendering the standard blocks menu while closing, as described in the second bullet point of https://github.com/WordPress/gutenberg/pull/5794#issuecomment-376059920. Perhaps there is a way to solve that issue without using local state though -- I'll think about it.